### PR TITLE
Drawer add support for minWidth

### PR DESCRIPTION
**Pull request by M-i-k-e-l**

### CODE REVIEW

File: src/interactableComponents/drawer/newDrawer.js [hunk 1]
```diff
@@ -53,6 +53,10 @@ export default class NewDrawer extends BaseComponent {
      * Whether to give the items equal width (the max width)
      */
     equalWidths: PropTypes.bool,
+    /**
+     * Set a different minimum width
+     */
+    minWidth: PropTypes.number,
```
Comment (ethanshar) [inline src/interactableComponents/drawer/newDrawer.js]: I would change to `itemsMinWidth` to keep it similar to other props (`itemsTintColor`, `itemsIconSize`, etc..)

File: src/interactableComponents/drawer/newDrawer.js [hunk 2]
```diff
@@ -118,8 +122,8 @@ export default class NewDrawer extends BaseComponent {
 
   // eslint-disable-next-line react/prop-types
   renderAction = ({item, index, progress, itemsCount}) => {
-    const {itemsTintColor, itemsIconSize, itemsTextStyle} = this.getThemeProps();
-
+    const {itemsTintColor, itemsIconSize, itemsTextStyle, minWidth} = this.getThemeProps();
+    const width = _.max([item.width, minWidth]);
```
Comment (ethanshar) [inline src/interactableComponents/drawer/newDrawer.js]: no need for that, react native's style support the `minWidth` prop. 

Also, If the user doesn't pass a `width` to an item but does pass the `minWidth` prop, the item width will be fixed and wouldn't stretch according to its content.

File: src/interactableComponents/drawer/newDrawer.js (hunk 1 above)
Comment (M-i-k-e-l) [reply src/interactableComponents/drawer/newDrawer.js]: Do we want all items to have the same minimum width? - Or maybe add item.minWidth?

Comment (M-i-k-e-l) [reply src/interactableComponents/drawer/newDrawer.js]: Done

File: src/interactableComponents/drawer/newDrawer.js (hunk 2 above)
Comment (M-i-k-e-l) [reply src/interactableComponents/drawer/newDrawer.js]: Done

**Pull request merged**